packages/sonarwhal/src/lib/cli/rules/new-rule.ts

```diff
@@ -29,30 +28,27 @@ const copyCommonFiles = async (destination: string) => {
 };
 
 const generateRuleFiles = async (destination: string, data) => {
-    const commonFiles = [{
-        destination: path.join(destination, '.sonarwhalrc'),
-        path: path.join(__dirname, TEMPLATE_PATH, 'rule-config.hbs')
-    },
-    {
-        destination: path.join(destination, 'README.md'),
-        path: path.join(__dirname, TEMPLATE_PATH, 'rule-doc.hbs')
-    },
-    {
-        destination: path.join(destination, 'package.json'),
-        path: path.join(__dirname, TEMPLATE_PATH, 'rule-package.hbs')
-    },
-    {
-        destination: path.join(destination, 'src', `index.ts`),
-        path: path.join(__dirname, TEMPLATE_PATH, 'rule-index.hbs')
-    }];
+    const commonFiles = [
+        {
+            destination: path.join(destination, 'README.md'),
```

Comment:
aren't we copying the `.sonarwhalrc` anymore?

Reply:
I'm not sure, @alrra voted no yesterday. Doesn't `.sonarwhalrc` come with `sonarwhal` now?

Reply:
@sarvaje Why do you think rules need a `.sonarwhalrc` file?

>  Doesn't .sonarwhalrc come with sonarwhal now?

Yes, if missing a `.sonarwhalrc` should be created based on the user's choices in the init process.

Reply:
> @sarvaje Why do you think rules need a .sonarwhalrc file?

Because I was thinking in an external rule, and we use to create the .sonarwhalrc file with the new rule preconfigured.

If you think that it is not necessary, is ok, but then I think we should show a message or something to the user to start using his new rule.